src/styles/main.css

```diff
@@ -1185,3 +1229,28 @@ a:focus-visible,
   color: #a0aab5;
   font-weight: 400;
 }
+
+/* Info and Empty State Messages */
+.info-message {
+  padding: 1rem;
+  background: #dbeafe;
+  border-left: 4px solid #3b82f6;
+  border-radius: 4px;
+  margin: 1rem 0;
+}
+
+.info-message p {
+  margin: 0;
+  color: #1e40af;
+}
+
+.empty-state {
+  padding: 2rem;
+  text-align: center;
+  color: #6b7280;
+}
+
+.empty-state p {
+  margin: 0;
+  font-size: 1rem;
+}
```

Comment:
This is starting to become a really large file. Should we break it into pieces? What's the best way for us to manage styles as this application grows?